.github/workflows/style.yml

```diff
@@ -1,15 +1,12 @@
 name: Style Checker
 
-# Runs my simple style(9) checker on any pushes or pull requests.  It could be
-# optimized by fetching the pull request head branch back to main revisions and
-# running on that. That would reduce the run time from 3-4 minutes down to 30-40
-# seconds. Getting the right series of clone + fetches to get that iteratively
-# is proving elusive, so optimizations welcome.
+# Runs my simple style(9) checker on pull requests.
 
 on:
   pull_request: # maybe pull_request_target
     branches: [ main ]
     types: [ opened, reopened, edited, synchronize ]
+    paths: [ '**.c', '**.cc', '**.h', '**.hh' ]
```

Comment:
I think `checkstyle9.pl` lints/checks the following source file types:
https://github.com/freebsd/freebsd-src/blob/6749f059a586ff62f2909134c4f4599c82fabc5e/tools/build/checkstyle9.pl#L15

Reply:
We need .S at least in the github path statement.

Reply:
> I think `checkstyle9.pl` lints/checks the following source file types:
> 
> https://github.com/freebsd/freebsd-src/blob/6749f059a586ff62f2909134c4f4599c82fabc5e/tools/build/checkstyle9.pl#L15

Hm, maybe we should add `cc` and `hh` here. Though, I don't know what the style rules are with C++ specific things (do we have any?).

I'll only add `.S`, as Warner suggested, since we don't really enforce a style on the others.